modules/cnf-topology-aware-lifecycle-operator-apply-policies.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="talo-apply-policies_{context}"]
+= Applying update policies to managed clusters
+
+The Topology Aware Lifecycle Operator remediates a set of managed policies in the specified clusters. Then, the Operator makes enforced copies of the managed {rh-rhacm} policies. Each copied policy has its own corresponding {rh-rhacm} placement rule and {rh-rhacm} placement binding. One by one, the Operator adds each cluster in the current batch to the placement rule that corresponds with the applicable managed policy. After a batch is updated, all clusters are removed from the placement rules associated with the copied policies. Then, the update of the next batch starts.
+
+.Prerequisites
+
+* Install the Topology Aware Lifecycle Operator.
+* Provision one or more managed clusters using ZTP.
+* Log in as a user with `cluster-admin` privileges.
+* Create an {rh-rhacm} policy in the Hub cluster.
+
+.Procedure
+
+. Create the `ClusterGroupUpgrade` CR, and then save the YAML in the `cgu-cr.yaml` file:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: du-upgrade-4915
+  namespace: ztp-group-du-sno
+spec:
+  deleteObjects: true
+  clusters: <1>
+  - cnfdb1
+  - cnfdb2
+  enable: true
+  managedPolicies:
+  - du-upgrade-platform-upgrade <2>
+  remediationStrategy:
+    maxConcurrency: 2 <3>
+    timeout: 240 <4>
+----
+<1> The list of clusters to update.
+<2> The name of the policy to apply.
+<3> The `maxConcurrency` field signifies the number of clusters updated at the same time.
+<4> The update timeout in minutes.
+
+. Create the `ClusterGroupUpgrade` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f cgu-cr.yaml
+----
+
+.Verification
+
+. Check if the `ClusterGroupUpgrade` CR was created in the Hub cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get cgu-cr -A -w
+----
++
+.Example output
++
+[source,terminal]
+----
+NAMESPACE          NAME              AGE
+ztp-group-du-sno   du-upgrade-4915   5s
+ztp-install        cnfdb1            4d5h
```

Comment:
Remove this line and the one below. These are older CGUs that were not created by applying the yaml above.